src/modules/api.js

```diff
@@ -0,0 +1,23 @@
+// const api1 = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+const api2 = 'https://api.tvmaze.com/shows';
+// styling
+
+const navDiv = document.getElementById('nav-div');
+const mediaStyle = (x) => {
```

Comment:
instead of x, if you used any descriptive name. It will be helpful for the team to read it clearly. 😄